sbin/Makefile.am

```diff
@@ -55,12 +28,29 @@ if ENABLE_VNETBUILD
 scripts += vnetbuild
 endif
 
-sbin_SCRIPTS = $(scripts)
+CLEANFILES = install.config
 
-EXTRA_DIST = \
-	commands.sed.in \
-	$(scriptsin) \
-	$(inclib_DATA)
+inclib_DATA = \
+	functions.common \
+	install.config \
+	$(NULL)
+
+inclib_SCRIPTS = $(scripts)
 
-uninstall-local:
+EXTRA_DIST = \
+	functions.common \
+	install.config.in \
+	$(scripts) \
+	$(NULL)
+
+install-exec-hook:
+	$(MKDIR_P) $(DESTDIR)$(sbindir)
+	for i in $(scripts); do \
+		$(RM) -f $(DESTDIR)$(sbindir)/$$i; \
+		$(LN_S) $(DESTDIR)$(inclibdir)/$$i $(DESTDIR)$(sbindir); done
+
+uninstall-hook:
+	for i in $(scripts); do \
+		$(RM) -f $(DESTDIR)$(sbindir)/$$i; done
 	@-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(inclibdir)
+	@-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(sbindir)
```

Comment:
This highlights a question I had.

It won't remove an existing directory if it has files so I think it is safe no? But the reason I put it in was I was testing `make install prefix="$HOME/firehol" and whilst the directory gets created automatically it would be left behind on uninstall.

Is there a preferred way to handle this or is the advice just to live with it (I doubt this is the only example and I guess it could get quite complex trying to handle all cases).